x-pack/plugins/cases/server/connectors/cases/cases_connector.ts

```diff
@@ -38,5 +97,300 @@ export class CasesConnector extends SubActionConnector<
     throw new Error('Method not implemented.');
   }
 
-  public async run() {}
+  public async run(params: CasesConnectorRunParams) {
```

Comment:
this is similar to `execute` function from stack_connectors plugin, right?

Reply:
Correct! This function will get executed when the action fires.